released/packages/coq-bits/coq-bits.1.1.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "anton.a.trunov@gmail.com"
+
+homepage: "https://github.com/coq-community/bits"
+dev-repo: "git+https://github.com/coq-community/bits.git"
+bug-reports: "https://github.com/coq-community/bits/issues"
+license: "Apache-2.0"
+
+synopsis: "A bit vector library"
+description: """
+A formalization of bitset operations in Coq and the corresponding
+axiomatization and extraction to OCaml native integers"""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" {(>= "8.10" & < "8.14~")}
+  "coq-mathcomp-algebra" {(>= "1.12" & < "1.13~")}
+  "ocamlbuild" {with-test}
```

Comment:
```suggestion
```
I don't see the point of adding a test-only dependency if you don't add a `run-test` clause as well. See for example the coq-equations and coq-simple-io packages for how this is done.

Reply:
Yes, I know how it's done, it's just not practical to install it and test at the same time because testing takes several hours. But I sometimes install packages' dependencies as `opam install foo --deps-only --with-test` and then experiment with `foo`'s source code given that all the deps are already in place. Maybe there is a better alternative to my workflow, though. Anyways, I remove the `ocaml` dependency.